ext/bcmath/methodsynopses/bcadd.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<methodsynopses>
+
+  <!-- bcadd -->
+  <methodsynopsis>
+   <modifier>public</modifier><type>string</type><methodname>bcadd</methodname>
```

Comment:
Should not have modifiers on free-standing functions.

Reply:
Fixed!